src/components/Menu/TopCodeMenu.js

```diff
@@ -26,7 +26,7 @@ const TopCodeMenu = ({ languages }) => {
               <LanguageButton language={language} />
             </MenuItem>
           ))}
-        </HorizontalCodeMenu>
+        </TopHorizontalMenu>
       ) : null}
```

Comment:
I know it is not your code, but `<></>` is better than `null` and will type better when we move to TS